com.eminfedar.vaktisalah-gtk.json

```diff
@@ -0,0 +1,101 @@
+{
+    "id": "com.eminfedar.vaktisalah-gtk",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "vaktisalah-gtk",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "libsass",
+            "buildsystem":"meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://github.com/lazka/libsass.git",
+                    "branch": "meson"
+                }
+            ]
+        },
+        {
+            "name": "sassc",
+            "buildsystem":"meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://github.com/lazka/sassc.git",
+                    "branch": "meson"
+                }
+            ]
+        },
+        {
+            "name": "libadwaita",
+            "buildsystem":"meson",
+            "config-opts" : [
+                "-Dvapi=false",
+                "-Dexamples=false",
+                "-Dtests=false"
+            ],
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://gitlab.gnome.org/GNOME/libadwaita.git",
+                    "tag": "1.3.2"
+                }
+            ]
+        },
+        {
+            "name": "blueprint-compiler",
```

Comment:
you need a cleanup rule to remove this build-only package completely

Reply:
`"cleanup": ["*"]`